state/filesystem.go

```diff
@@ -1002,6 +1002,18 @@ func validateAddExistingFilesystem(
 				backingVolume.Size, info.Size,
 			)
 		}
+		// Verify volumeid have not been registered.
+		vol, err := sb.GetVolumeByVolumeId(backingVolume.VolumeId)
```

Comment:
This isn't part of this work right?
There should be no state changes in this PR since the only changes are to modify the k8s import logic in the provider

Reply:
It's used in the import filesystem flow. I think we need this one.